client/src/views/client-area/statistics/CustomOrderStatisticsPage.vue

```diff
@@ -0,0 +1,719 @@
+              <ion-text class="users-cards-list-item-text__text subtitles-sm">
+                {{
+                  $msTranslate(
+                    organizationStats.adminUsersDetail.revoked > 1 ? 'clientArea.statistics.admins' : 'clientArea.statistics.admin',
+                  )
+                }}
+              </ion-text>
+            </div>
+          </ion-card>
+          <ion-card class="users-cards-list-item">
+            <div class="users-cards-list-item-icons">
+              <ion-icon
+                class="users-cards-list-item-icons__main"
+                :icon="person"
+              />
+              <ion-icon
+                class="users-cards-list-item-icons__secondary"
+                :icon="pencil"
+              />
+            </div>
+            <div class="users-cards-list-item-text">
+              <ion-text class="users-cards-list-item-text__number title-h1">
+                {{ organizationStats.standardUsersDetail.revoked || 0 }}
+              </ion-text>
+              <ion-text class="users-cards-list-item-text__text subtitles-sm">
+                {{
+                  $msTranslate(
+                    organizationStats.standardUsersDetail.revoked > 1
+                      ? 'clientArea.statistics.standards'
+                      : 'clientArea.statistics.standard',
+                  )
+                }}
+              </ion-text>
+            </div>
+          </ion-card>
+          <ion-card class="users-cards-list-item">
+            <div class="users-cards-list-item-icons">
+              <ion-icon
+                class="users-cards-list-item-icons__main"
+                :icon="person"
+              />
+              <ion-icon
+                class="users-cards-list-item-icons__secondary"
+                :icon="arrowForward"
+              />
+            </div>
+            <div class="users-cards-list-item-text">
+              <ion-text class="users-cards-list-item-text__number title-h1">
+                {{ organizationStats.outsiderUsersDetail.revoked || 0 }}
+              </ion-text>
+              <ion-text class="users-cards-list-item-text__text subtitles-sm">
+                {{
+                  $msTranslate(
+                    organizationStats.outsiderUsersDetail.revoked > 1
+                      ? 'clientArea.statistics.outsiders'
+                      : 'clientArea.statistics.outsider',
+                  )
+                }}
+              </ion-text>
+            </div>
+          </ion-card>
+        </div>
+      </div>
+      <!-- storage -->
+      <div class="storage">
+        <ion-text class="storage__title title-h2">{{ $msTranslate('clientArea.statistics.titles.storage') }}</ion-text>
+        <div class="storage-data">
+          <div class="storage-data-global">
+            <ion-text
+              class="storage-data-global__total title-h1"
+              :class="useStoragePercentage >= 100 ? 'overused' : ''"
```

Comment:
It can be written as `:class="{ 'overused': useStoragePercentage >= 100}"`, which I find a little cleaner.